UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamageDiscriptors.cs

```diff
@@ -23,11 +23,35 @@ public partial class BodyPart
 		private string pireceDamageDescOnLARGE= "{readableName} suffers from a Ruptured Cavity.";
 		[SerializeField] private string internalDamageDesc	 = "This {readableName} is suffering from internal damage.";
 		[SerializeField] private string externalBleedingDesc = "This {readableName} is bleeding due to an open wound.";
+		[SerializeField, NaughtyAttributes.EnableIf("CanBeBroken")] private string BoneBrokenDesc = "This {readableName} is suffering Hairline Fracture.";
+		[SerializeField, NaughtyAttributes.EnableIf("CanBeBroken")] private string BoneFracturedDesc = "This {readableName} is suffering Compound Fracture. It is completely snapped in half.";
 
 		public string GetFullBodyPartDamageDescReport()
 		{
 			string report = $"Analyizing damage report for {BodyPartReadableName}.. \n";
 
+			if (CanBeBroken)
+			{
+				CheckIfBroken();
+				report += "[Fracture Level]\n";
+				if (IsBroken)
+				{
+					return TranslateTags(report + BoneBrokenDesc);
```

Comment:
use interpolation over concatenation whenever it is possible

Reply:
wdym?

Reply:
$"string {variable}"

Reply:
I think I added what you wanted.

Reply:
Using a string builder now, this good?